index.css

```diff
@@ -1,3 +1,4 @@
+@import "@nib-styles/v2-icons";
 
 @keyframes v2-alert-pop-out {
```

Comment:
Not sure. There weren't any notations on the UI design that Chris gave me. I'm following him up on it.